backend/config.go

```diff
@@ -87,18 +87,13 @@ func (ft FeatureToggles) IsEnabled(f string) bool {
 }
 
 type Proxy struct {
-	clientCfg proxy.ClientCfg
-}
-
-func (pc Proxy) ClientConfig() proxy.ClientCfg {
```

Comment:
Realized I don't need to expose this since this will likely remain internal and not of any use to developers